pliers/extractors/audio.py

```diff
@@ -488,4 +494,140 @@ class PercussiveExtractor(LibrosaFeatureExtractor):
     For details on argument specification visit:
     https://librosa.github.io/librosa/effect.html.'''
 
+
     _feature = 'percussive'
+
+
+YAMNET_INSTALL_MESSAGE = '''
```

Comment:
@tyarkoni any other thoughts on how to install?

this is not a python package, so pip can't be used. @rbroc talked about this, and we came up with adding this folder to the python path. You can also copy the contents to `site-packages`. Both work, but are at bit janky since it doesn't even have an `__init__.py` file.

An alternative is to provide a helper function is `support` but it seems like overkill and maybe a user problem given instructions are given?  That said, if a user choses to make it a package and install it properly, the way we import it here won't work, as it expects the `yamnet` directory to be part of the path (rather than a package *in* the path).

Reply:
added `setup_yamnet` to support which should do the job. Also makes installation on travis smoother.